provisioning/miniprov/hare_mp/main.py

```diff
@@ -93,7 +93,7 @@ def setup_logging(url) -> None:
 
     create_logger_directory(log_dir)
 
-    console = logging.StreamHandler(stream=sys.stderr)
+    console = logging.StreamHandler(stream=sys.stdout)
```

Comment:
I think instead of just changing stderr to stdout, where all the error as well as regular logs will be re-directed to just one stream, we need to have a way to log errors/exceptions to stderr while regular logs to stdout.
This can be done by having 2 logging streams,  1 for stderr and other for stdout.
A single wrapper class around these logging streams can be created to log messages to the respective streams.

Reply:
@mssawant I think it's an overkill.
As I had written earlier, there is no specific rule that writing to stderr means a failure. It means that the application is free to choose where to write. A failure is a failure if and only if the exit code is non-zero, everything else depends on use cases of the particular application.

What you suggest is a nice to have but I'm pretty sure that it requires 1-2 days of investigation how to tune the logging subsystem in that fine manner. Is it worthwhile?

Reply:
In that case we need to make sure that every exception is handled such that a non-zero error code is returned to the caller, it should not happen that an exception is raised but the operation was a success. If we guarantee that, then I am okay with the current implementation.

Reply:
I have tested the deployment with all the changes and populated the result on this Jira : https://jts.seagate.com/browse/EOS-26779